platform/android/godot_android.cpp

```diff
@@ -789,7 +789,7 @@ void android_main(struct android_app* state) {
 	     ASENSOR_TYPE_ACCELEROMETER);
      // assuming the Android API is uniform
```

Comment:
Could you also remove this misleading comment, now that we're established that the Android API is not uniform, but also removed the need for assumptions by checking the API reference?